<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recommend</class>
 <widget class="QWidget" name="recommend">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>您的菜单请查收！</string>
  </property>
  <widget class="QPushButton" name="suButton_1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>240</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;宋体&quot;;
color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="suButton_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>170</y>
     <width>240</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;宋体&quot;;
color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="suButton_3">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>170</y>
     <width>240</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;宋体&quot;;
color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>健康绿色</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hunButton_1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>240</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;宋体&quot;;
color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="hunButton_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>350</y>
     <width>240</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;宋体&quot;;
color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sulabel_1">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>170</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>燕园美食</string>
   </property>
  </widget>
  <widget class="QLabel" name="sulabel_2">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>170</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sulabel_3">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>170</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="hunsuButton_1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>530</y>
     <width>240</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;宋体&quot;;
color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="hunButton_3">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>350</y>
     <width>240</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;宋体&quot;;
color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>蛋白营养</string>
   </property>
  </widget>
  <widget class="QLabel" name="hunlabel_3">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>350</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hunlabel_2">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>350</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hunlabel_1">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>350</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>540</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>荤素搭配</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hunsuButton_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>530</y>
     <width>240</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;宋体&quot;;
color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="hunsuButton_3">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>530</y>
     <width>240</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;宋体&quot;;
color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>720</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>均衡膳食</string>
   </property>
  </widget>
  <widget class="QPushButton" name="taoButton_3">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>710</y>
     <width>240</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;宋体&quot;;
color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="taoButton_2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>710</y>
     <width>240</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;宋体&quot;;
color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="taoButton_1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>710</y>
     <width>240</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;宋体&quot;;
color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hunsulabel_3">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>530</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hunsulabel_2">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>530</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hunsulabel_1">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>530</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="taolabel_1">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>710</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="taolabel_2">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>710</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="taolabel_3">
   <property name="geometry">
    <rect>
     <x>1540</x>
     <y>710</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1670</x>
     <y>30</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
